repository/Cormas-Core/CMSimManager.class.st

```diff
@@ -2973,7 +2978,7 @@ CMSimManager >> enableBackward: bool [
 
 { #category : #time }
 CMSimManager >> finalTime [
-	^ finalTime ifNil: [ finalTime := self cormasModel class defaultFinalStep ifNil: [ nil]  ]
+	^ finalTime ifNil: [ finalTime := self defaultFinalTime]
```

Comment:
I will called it ``finalStep`` and ``defaultFinalStep``, because these are steps and not really time.

Reply:
I thought of that, too, but there are many senders of finalTime here and there. I think changing this makes too big impact. Maybe another issue?

Reply:
I see only 10 senders. Do you think this is too much?

Reply:
You are right, let's do the change of name in another PR.

Reply:
My point is that some modelers use finalTime: to set up their simulation steps. At least we need to deprecate it first before deleting it.

Reply:
ok